### DISCUSSION TITLE
Can we add Download button for Web ?

### DISCUSSION THREAD

**Discussion by Gujjugaming2k** [DonutWare/Fladder#193] [Ideas]
### Summary

Can we add Download button for Web ?

same like play if we are clicking on play button its fetching stream link and same for download as well like copy stream link, download button

### Basic Example

same like play if we are clicking on play button its fetching stream link and same for download as well like copy stream link, download button

### Drawbacks

na

**Comment by PartyDonut** [comment]
Added in #209
